cmd/agents/app/main.go

```diff
@@ -106,9 +107,9 @@ func main() {
 		os.Exit(1)
 	}
 	if err = (&controllers.ServiceClaimReconciler{
-		Client: mgr.GetClient(),
-		Scheme: mgr.GetScheme(),
-	}).SetupWithManager(mgr); err != nil {
+		ServiceClaimReconciler: sccontrollers.ServiceClaimReconciler{Client: mgr.GetClient(),
```

Comment:
Why are we not calling the controller created on the agent app, and importing the controller defined in primaza control plane?

Reply:
If you look at the line [no. 109](https://github.com/primaza/primaza/pull/111/files#diff-cf4eaab8f5c6a392cc61fc52e5b38642592ba9454e9509aeebed7725cd15b72dR109), it's calling the agent app. The struct embedding the main ServiceClaimReconciler. Line no. 110 shows that.